mycpp/gc_builtins.cc

```diff
@@ -291,6 +291,18 @@ bool are_equal(Tuple2<Str*, int>* t1, Tuple2<Str*, int>* t2) {
   return result;
 }
 
+bool are_equal(Tuple2<int, int>* t1, Tuple2<int, int>* t2) {
+  return t1->at0() == t2->at0() && t1->at1() == t2->at1();
+}
+
+bool keys_equal(Tuple2<int, int>* t1, Tuple2<int, int>* t2) {
```

Comment:
I feel like there is some template refactoring to simplify the keys_equal / are_equal stuff, but let's worry about that separately ...